package.json

```diff
@@ -76,11 +76,11 @@
     "remark-cli": "^4.0.0",
     "remark-validate-links": "^6.0.0",
     "rimraf": "^2.4.3",
-    "rollup": "^0.47.6",
+    "rollup": "^0.48.2",
     "rollup-plugin-buble": "^0.15.0",
     "rollup-plugin-commonjs": "^8.1.0",
     "rollup-plugin-istanbul": "^1.1.0",
-    "rollup-plugin-multi-entry": "^2.0.1",
+    "rollup-plugin-multi-entry": "3.0.0",
```

Comment:
Waiting for this version to be released, which I guess it will once rollup/rollup-plugin-multi-entry#22 is merged.

Reply:
PR has been superseded by https://github.com/rollup/rollup-plugin-multi-entry/pull/23. Waiting for that one now.